frontend/app/modules/Create/create-issue-controller.js

```diff
@@ -0,0 +1,19 @@
+(function(issues) {
+  'use strict';
+  angular
+    .module('issue-tracker.Create')
```

Comment:
Modules should be lowercased - perhaps a better name would be `issues-tracker.issues`